float32 vigilance
int32 indexCategory
---
